### PR TITLE
skip AOTI UTs on Windows.

### PR DESCRIPTION
**Pull request by xuhancn**
Skip UTs depends on Windows AOTI, due to Windows not support AOTI so far.

cc @peterjc123 @mszhanyi @skyline75489 @nbcsm @iremyux @Blackhex @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10 @voznesenskym @penguinwu @EikanWang @Guobing-Chen @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

File: test/inductor/test_torchinductor.py [hunk 1]
```diff
@@ -1680,6 +1680,7 @@ def flip(x):
         actual = _run_and_assert_no_indirect_indexing(self, flip_opt, x)
         self.assertEqual(expect, actual)
 
+    @skipIfWindows(msg="AOTI not support Windows: aoti_torch_check failed.")
```
[the remark this answers is not in the document]
Comment (xuhancn) [reply test/inductor/test_torchinductor.py:1683]: https://github.com/pytorch/pytorch/blob/20af56d4359c3f5fed2e8f94e111a8502f2ebeb3/torch/csrc/inductor/aoti_torch/c/shim.h#L47 @desertfire it is comment the export declare as not support aoti. So it reporting link issue. 
Or we enable the export declare on Windows?

[the remark this answers is not in the document]
Comment (jgong5) [reply test/inductor/test_torchinductor.py:1683]: What are the remaining gaps if we export it for Windows now?

**Review by jansel — APPROVED** [review]

**Comment by xuhancn** [comment]
Close by new PR: https://github.com/pytorch/pytorch/pull/140030 CC: @desertfire, @jgong5

**Pull request closed without merging**